complete-application/src/index.css

```diff
@@ -159,13 +173,7 @@ body {
   min-width: 100%;
```

Comment:
we can just let the content fill the width. If min-width is specified here it can overflow since box-sizing is default to content-box https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

not a very big deal though.

<img width="569" alt="image" src="https://github.com/FusionAuth/fusionauth-quickstart-javascript-react-web/assets/49047386/46c46b80-d0b6-46ba-b186-2e96f604be8c">

Reply:
Fixed this as well, thank you